web3/.env.example

```diff
@@ -0,0 +1,2 @@
+PRIVATE_KEY='YOUR KEY'
+SEPOLIA_RPC_URL=https://eth-sepolia.g.alchemy.com/v2/pOXRE5q4Mu5BJj-3sw4DaIOErWcnsI1Q 
```

Comment:
where did you get this rpc url?

Reply:
You can create your own RPC node on Alchemy or use the publicly available nodes. Your choice.